custom_components/plugwise_usb/select.py

```
@@ -0,0 +1,151 @@
+                "No %s select data for %s",
+                str(self.entity_description.node_feature),
+                self._node_info.mac,
+            )
+            return
+
+        current_option = getattr(
+            data,
+            self.entity_description.api_attribute,
+        )
+        self._current_option = current_option.name
+        self._attr_current_option = current_option.name 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)